src/app/shared/recommendation-card/recommendation-card.component.ts

```diff
@@ -24,7 +25,12 @@ export class RecommendationCardComponent  implements OnInit {
   }
 
   redirect(){
-    this.router.navigate(["mi-details",this.cardData._id ])
+    if(this.type === 'journey'){
+      this.router.navigate([])
+    }
```

Comment:
what is the purpose of this.router.navigate([]) ?

Reply:
this will redirect to the view story page.